pkgs/tools/networking/openapi-generator-cli/default.nix

```diff
@@ -35,10 +35,11 @@ let this = stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [ binaryBytecode ];
```

Comment:
I think you might've missed one in `changelog`

Reply:
well spotted ty ! Fixed.